test/unit/template-parts.spec.js

```diff
@@ -0,0 +1,579 @@
+/* Adapted from https://github.com/dy/template-parts - ISC - Dmitry Iv. */
+
+import { assert } from '@open-wc/testing';
+import { TemplateInstance, AttrPart, AttrPartList } from '../../src/js/utils/template-parts.js';
+
+const test = it;
+const is = assert.deepEqual;
+const any = (a, b) => assert.include(b, a);
+
+test('create: applies data to templated text nodes', () => {
```

Comment:
nitpick(blocking): Unless we expect to regularly update this via a copy-paste from the original `template-parts` repo, I'd recommend refactoring this to conform to our other test modules (aka wrap in a `describe` and use our standard assertions.